### ISSUE TITLE
v3.1.0 - Array.prototype.forEach called on null or undefined in configParser.js:154

### ISSUE THREAD

**Issue by flynetworks** [angular/protractor#2925]
If i use multiCapabilities the following error comes up:

[10:13:38][Step 4/5] TypeError: Array.prototype.forEach called on null or undefined
[10:13:38][Step 4/5]     at forEach (native)
[10:13:38][Step 4/5]     at Function.ConfigParser.getSpecs (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/lib/configParser.js:154:27)
[10:13:38][Step 4/5]     at new TaskScheduler (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/lib/taskScheduler.js:32:20)
[10:13:38][Step 4/5]     at /data/buildagent/work/e0815e63242a3116/node_modules/protractor/lib/launcher.js:173:21
[10:13:38][Step 4/5]     at _fulfilled (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:834:54)
[10:13:38][Step 4/5]     at self.promiseDispatch.done (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:863:30)
[10:13:38][Step 4/5]     at Promise.promise.promiseDispatch (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:796:13)
[10:13:38][Step 4/5]     at /data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:604:44
[10:13:38][Step 4/5]     at runSingle (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:137:13)
[10:13:38][Step 4/5]     at flush (/data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:125:13)
[10:13:38][Step 4/5] >> /data/buildagent/work/e0815e63242a3116/node_modules/protractor/node_modules/q/q.js:155

I was able to fix that by wrapping the code within an if condition like this:
![image](https://cloud.githubusercontent.com/assets/384104/12944778/a0b7544e-cfe9-11e5-8cf6-5b04e17c88e6.png)

**Comment by carlos-granados** [comment]
I also get this problem and I am not using multicapabilities. My protractor config file:

```
exports.config = {
    baseUrl: 'https://staging-i6.localhost/app_test.php',
    directConnect: true,
    framework: 'custom',
    frameworkPath: require.resolve('protractor-cucumber-framework'),
    allScriptsTimeout: 60000,
    getPageTimeout: 60000,
    rootElement: 'body',
    capabilities: {
        browserName: 'chrome',
        'chromeOptions': {
            args: [ 'ignore-certificate-errors=true' ]
        },
        specs: [
            'jsfeatures/*.feature'
        ]
    },
    cucumberOpts: {
        require: 'jsfeatures/steps/*_steps.js',
        //uncomment the follwing line if you want to only run the features tagged @run
        //tags: ["@run"],
        format: 'pretty'
    }
};
```

**Comment by juliemr** [comment]
Thanks for the quick error reports. This is fixed in release 3.1.1, which is out now!
